node/builder_miner.go

```diff
@@ -4,6 +4,7 @@ import (
 	"errors"
 	"time"
 
+	"github.com/filecoin-project/go-fil-markets/shared_testutil"
```

Comment:
(that's a weird package name for a DAGStore)

Reply:
@magik6k This is a mocked out DAG Store. See the TODO. We will soon be integrating the [actual DAGStore]([url](https://github.com/filecoin-project/dagstore)).

Reply:
Having said that, even this has been fixed.